peacebot/core/plugins/Miscellaneous/misc.py

```diff
@@ -1,24 +1,25 @@
+import random
 from datetime import datetime, timedelta
 
 import hikari
 import lightbulb
 
-import peacebot.core.utils.helper_functions as hf
 from peacebot.core.utils.embed_colors import EmbedColors
+from peacebot.core.utils.helper_functions import convert_time, error_handler
 
-from . import convert_time, fetch_scheduler, send_remainder
+from . import fetch_scheduler, send_remainder
 
 misc_plugin = lightbulb.Plugin("Misc", "Miscellaneous Commands for the Bot")
 
 
 @misc_plugin.command
+@lightbulb.option("target", "The target of this command", type=hikari.Member)
 @lightbulb.add_cooldown(1, 5, lightbulb.UserBucket)
-@lightbulb.option("member", "Get member", type=hikari.Member, required=False)
-@lightbulb.command("avatar", "View a member's avatar", aliases=["av"])
-@lightbulb.implements(lightbulb.PrefixCommand, lightbulb.SlashCommand)
-@hf.error_handler()
+@lightbulb.command("avatar", "View a member's avatar")
+@lightbulb.implements(lightbulb.SlashCommand, lightbulb.PrefixCommand)
+@error_handler()
 async def avatar(ctx: lightbulb.Context) -> None:
-    member: hikari.Member = ctx.options.member or ctx.member
+    member: hikari.Member = ctx.options.target
```

Comment:
Why are we not allowing default target to be the author themselves?

Reply:
`Context Menu commands` are right around the corner, so there is no point in allowing the author to be the default target.
All this has been done to smoothen the transition to Context Menu Commands.

Reply:
Yep, you have a point.